content/docs/guides/continuous-delivery/octopus-deploy.md

```diff
@@ -0,0 +1,118 @@
+
+## Create and configure a project
+
+A [deployment process](https://octopus.com/docs/deployment-process) is comprised of the following steps:
+
+- Configure Project Variables
+- Create the Process
+
+### Configure Project Variables
+You can configure your AWS Account and Azure Subscription credentials as project variables using the built-in `AWS Account` and `Azure Subscription` variable types. Before you can do that, though, you will need to add them to the **Accounts** section under **Infrastructure** in your Octopus instance.
+
+For other cloud providers, you will need to add the account credentials directly as a project variable using the appropriate built-in variable type.
+
+Variables can be scoped right down to the environment that can access them. For sensitive strings, be sure to set the right scopes so that the account credentials are not accidentally used by the wrong environments.
+
+### Create the Process
+
+A process consists of the steps to execute in a project. The following sections outline the steps for configuring your process in order to use Pulumi to deploy infrastructure (not be confused with Octopus Infrastructure).
+
+#### Extract the code package
+
+- Click on the **Add Step** button in your project's **Process** section.
+- Click the **Referenced Packages** and choose the package that contains your Pulumi infrastructure app.
+  - **Package feed**: Choose the appropriate source for your package.
+  - **Package ID**: This is usually derived from your package, but enter a name of your choice. This ID will be used later.
+  - **Package Name**: Enter any name.
+  ![Add Package Reference](/images/docs/guides/continuous-delivery/octopus-deploy/package-reference.png)
+- Select the **Run a Script**, fill out the following fields as follows:
+  - **Script Source**: Inline source code
+  - **Inline Source Code**: (click the **Bash** radio button)
+  ```bash
+    # Remove any previously extracted packages
+    rm -rf /tmp/pulumi-app
+    # Use the pre-defined variable to get the extracted path for the package.
+    extractedPath=$(get_octopusvariable "Octopus.Action.Package[aws-typescript].ExtractedPath")
+    # Copy the extracted package contents to another directory that can be accessed by other steps.
+    cp -r "${extractedPath}/infrastructure/" /tmp/pulumi-app
+    cd /tmp/pulumi-app
+  ```
+
+#### Run Pulumi
+
+- Click on **Add Step** again and search for `Pulumi` from the **Community Library Steps**.
+- Fill out the configuration as appropriate for your stack.
+  - For example, in the following screenshot the stack configuration is overridden using a
+file that is created in a separate **Run a script** step.
+  - It is not required that you override stack configuration. It was done here purely for demonstrating
+  the options you have with the Pulumi Step Template.
+![Add Package Reference](/images/docs/guides/continuous-delivery/octopus-deploy/run-pulumi.png)
```

Comment:
Nit: Can we put a new line before the image? I wouldn't bet my life that some minor tweak to the Markdown rendering library we use might mistakenly try to put the image within the bulleted list or something.

Reply:
Done.